### PR TITLE
Change MNEMONIC var name for production & clean Hardhat file

### PR DESCRIPTION
**Pull request by zguesmi**

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -45,8 +45,6 @@ const v8Settings = {
     evmVersion: 'paris',
 };
 
-const zeroGasPrice = 0; // 0 Gwei. No EIP-1559 on Bellecour (Production sidechain).
```
Comment (jeremyjams) [inline hardhat.config.ts:48]: :pray:

File: hardhat.config.ts [hunk 2]
```diff
@@ -117,7 +115,7 @@ const config: HardhatUserConfig = {
             chainId: 1,
             url: process.env.MAINNET_NODE || '',
             accounts: {
-                mnemonic: process.env.MNEMONIC || '',
+                mnemonic: process.env.PROD_MNEMONIC || '',
             },
         },
         ropsten: {
```
Comment (jeremyjams) [inline hardhat.config.ts:121]: If this is a cleaning story, can you please remove:
- ropsten
- rinkeby
- kovan
:pray:

File: hardhat.config.ts [hunk 3]
```diff
@@ -70,26 +68,26 @@ const config: HardhatUserConfig = {
...
             }),
-            accounts: {
-                mnemonic: process.env.MNEMONIC || HARDHAT_NETWORK_MNEMONIC,
-            },
-            ...((isNativeChainType || isLocalFork) && bellecourNetworkConfig),
         },
         'external-hardhat': {
             ...defaultHardhatNetworkParams,
-            url: defaultLocalhostNetworkParams.url,
+            ...defaultLocalhostNetworkParams,
```
Comment (jeremyjams) [inline hardhat.config.ts:84]: Which issue did you encounter?

File: hardhat.config.ts (hunk 2 above)
Comment (zguesmi) [reply hardhat.config.ts]: Yes [Clean networks](https://github.com/iExecBlockchainComputing/PoCo/pull/53/commits/6c55580f0096578ffb4aab4cd4c7541bc971b94f)

Comment (jeremyjams) [reply hardhat.config.ts]: Thanks!

File: hardhat.config.ts (hunk 3 above)
Comment (zguesmi) [reply hardhat.config.ts:84]: No issue, just cleaner.

Comment (jeremyjams) [reply hardhat.config.ts:84]: OK nice :+1:

**Review by jeremyjams — APPROVED** [review]
Good thanks!

**Pull request merged**